components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -0,0 +1,109 @@
+                                  </span>
+                                  <span className="">Personal Account</span>
+                              </div>
+                          ),
+                          active: team === undefined,
+                          separator: true,
+                          link: `${location.pathname}?org=0`,
+                      },
+                  ]
+                : []),
+            ...(teams || [])
+                .map((t) => ({
+                    title: t.name,
+                    customContent: (
+                        <div className="w-full text-gray-400 flex flex-col">
+                            <span className="text-gray-800 dark:text-gray-300 text-base font-semibold">{t.name}</span>
+                            <span className="">
+                                {!!teamMembers[t.id]
+                                    ? `${teamMembers[t.id].length} member${teamMembers[t.id].length === 1 ? "" : "s"}`
+                                    : "..."}
+                            </span>
+                        </div>
+                    ),
+                    active: team?.id === t.id,
+                    separator: true,
+                    link: `${location.pathname}?org=${t.id}`,
+                }))
+                .sort((a, b) => (a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1)),
+            {
+                title: "Create a new organization",
+                customContent: (
+                    <div className="w-full text-gray-400 flex items-center">
+                        <span className="flex-1 font-semibold">New Organization</span>
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 14 14" className="w-3.5">
+                            <path
+                                fill="currentColor"
+                                fillRule="evenodd"
+                                d="M7 0a1 1 0 011 1v5h5a1 1 0 110 2H8v5a1 1 0 11-2 0V8H1a1 1 0 010-2h5V1a1 1 0 011-1z"
+                                clipRule="evenodd"
+                            />
+                        </svg>
+                    </div>
+                ),
+                link: "/orgs/new",
+            },
+        ],
+        [
+            user?.additionalData?.isMigratedToTeamOnlyAttribution,
+            userFullName,
+            team,
+            location.pathname,
+            teams,
+            teamMembers,
+        ],
+    );
+    const selectedEntry = entries.find((e) => e.active) || entries[0];
+    const classes =
+        "flex h-full text-base py-0 text-gray-500 bg-gray-50  dark:bg-gray-800 dark:text-gray-400 hover:bg-gray-100 dark:hover:bg-gray-700 dark:border-gray-700";
+    return (
+        <ContextMenu customClasses="w-64 left-0 pl-2" menuEntries={entries}>
+            <div className={`${classes} rounded-2xl pl-3`}>
+                <div className="py-1 pr-2 font-semibold">{selectedEntry.title!}</div>
```

Comment:
Can we add the class `whitespace-nowrap` to this div so that the org name doesn't wrap funny sometimes? 

We could also add a max-width if we need, and a tooltip to show the full name if we clip it.